### ISSUE TITLE
508 compliance issues (Accessibility) Reports page, the maps do not have any accessibility features

### ISSUE THREAD

**Issue by christopherwoodland** [Azure/api-management-developer-portal#1384]
## Bug description

Our client has noticed the following accessibility concerns and asked that we submit these concerns to the product team for resolution. These issues violate the 508 compliance, which the customer is bound to.  I am willing to aid in any way to bring these concerns to resolution. Please let me know if additional context or information is required to help move these concerns along.  
I can be reached internally via teams or email cwoodland@microsoft.com
Thank you in advance!

1. On the Reports page, the maps do not have any accessibility features. If you tab through the page, the maps do not receive focus.  The <svg> HTML element of each map is also missing either a descriptive <title> or <desc> as its first child element.
![image](https://user-images.githubusercontent.com/16299417/125957321-d9b8f02d-db0f-4908-be93-1a2020fd9f8e.png)

 #1383

**Comment by christopherwoodland** [comment]
@jdm4344  Perfect!  I will reference and close these issues.

#1374, #1375, #1378, #1379, #1380.
